sonic_platform_base/chassis_base.py

```diff
@@ -151,6 +151,9 @@ def is_modular_chassis(self):
             A bool value, should return False by default or for fixed-platforms.
             Should return True for supervisor-cards, line-cards etc running as part
             of modular-chassis.
+            For SmartSwitch platforms this should return True even if they are
```

Comment:
NIT: This statement is little confusing at first read. Should it be "For SmartSwitch platforms, this should return True even if they are fixed platforms, as they are treated like a modular chassis. The DPU cards in these systems are treated as line cards of a modular chassis."

Reply:
Fixed it:  "For SmartSwitch this should return False."